cypress/cypress/e2e/submariner_uninstall.cy.js

```diff
@@ -0,0 +1,38 @@
+/** *****************************************************************************
+ * Licensed Materials - Property of Red Hat, Inc.
+ * Copyright (c) 2021 Red Hat, Inc.
+ ****************************************************************************** */
+
+/// <reference types="cypress" />
+
+import { submarinerClusterSetMethods } from '../views/submariner/actions/submariner_actions'
+
+describe('submariner - uninstall validation', {
+    tags: ['@submariner', '@e2e'],
+    retries: {
+        runMode: 0,
+        openMode: 0,
+    }
+}, function () {
+    before(function () {
+        cy.setAPIToken()
+        cy.clearOCMCookies()
+        cy.login()
+        
+    })
+
+    it('verify that a cluster set with deployment of submariner is exists.', { tags: ['verify'] }, function () {
```

Comment:
Isn't we already has that test in "submariner_deployment_validation"?

I don't see it serves as dependency of the uninstall test.

Reply:
This is an initialization test I do in order to check that the relevant cluster set (which has the submariner deployment) is exists and also in order to enter this cluster set and start the uninstall test from this position. this test does not check the deployment of submariner.

Reply:
But as far as I understand, if submariner deployment will not exists, the test will fail and the uninstall test will fail as well.
Right?

That's a problem.
We don't want to fail a test in case there is nothing to test.

I suggest to the possibility of skipping the test in case the condition does not met.
For example, if no submariner deployment found, verify submariner test and uninstall submariner test should not be executed. They should be skipped.

Please, take a look at the example of clc team in test skip.
https://github.com/stolostron/clc-ui-e2e/blob/030a03e5d423d113095c9db88fca2a487db216ca/cypress/tests/clusters/managedClusters/reimportCluster.spec.js#L41
https://github.com/stolostron/clc-ui-e2e/blob/a091940538555c1db7b9fbe9a9ce76ccb3d79733/cypress/tests/clusters/centrallyManagedClusters/standalone/createClusterWithIntegratedAI.spec.js#L34
https://github.com/stolostron/clc-ui-e2e/blob/1ebdbe633e09f951fc4cccec0cd51d18662b4ce5/cypress/tests/clusters/managedClusters/localcluster.spec.js#L23
https://github.com/stolostron/clc-ui-e2e/blob/111ad4ebfce2316535762597db354c8c00444f72/cypress/tests/clusters/centrallyManagedClusters/hosted/createHyperShiftCluster.spec.js#L85


My only question regarding this is: when the skip is made, does the test in the report marked as skipped?

Reply:
Done